decidim-elections/app/models/decidim/elections/question.rb

```diff
@@ -22,6 +22,17 @@ class Question < ApplicationRecord
       def valid_max_selection?
         max_selections <= answers.count
       end
+
+      def total_votes
+        answers.sum(:votes)
+      end
+
+      def votes_calucations(answer_votes)
```

Comment:
I would rename the method to `votes_percentage`, so it is more clear what it returns

Reply:
True! I also saw that I wrote `calucations` instead of `calculations`. 😂🤦‍♀️ I'll change it!